### DISCUSSION TITLE
Trying to make a Template Switch which will be used as an entity for Person detection

### DISCUSSION THREAD

**Discussion by cpainchaud** [fwestenberg/reolink_dev#244] [Q&A] [answered]
currently we have one kind of switch for all kind of detection event with this integration:
![image](https://user-images.githubusercontent.com/6696638/128679713-fae8d61c-7407-4a1e-90f1-9a85a5bd9f07.png)
But this is not useful to show only "Person" detection events for example.

Has anyone tried using a Template switch like this (https://www.home-assistant.io/integrations/template/) or found a way to show only Person detection events?

**Comment by cpainchaud** [comment]
I have tried things like :

- platform: template
  sensors:
    cam_seine_person_detected:
      value_template: >-
        {{ state_attr('binary_sensor.cam_rive_motion', 'People') }}

i am getting close but I can't tell why it says state is unknown

**Comment by cpainchaud** [comment, marked as answer]
It was just a case sensivity issue!!

```platform: template
  sensors:
    cam_rive_person_detected:
      value_template: >-
        {{ state_attr('binary_sensor.cam_rive_motion', 'people') }}```

**Reply by rodakswe** [reply]
Does your attribute ever change to "True"?
I had it working with previous firmware version and my template sensor worked like a charm.
"Face" was not present and "People" was named "People state".
Now with the latest camera firmware that has "Face" and "People" the attribute never changes to "True" even if the motion is detected by the camera.

**Comment by cpainchaud** [comment]
I am having the issue. It's never switching to true.

**Reply by rodakswe** [reply]
What model of the camera do you have? I have restored my camera to a previous firmware to restore the "Person state" functionality.
I've also created an issue here: 
https://github.com/fwestenberg/reolink_dev/issues/245

Please comment on this issue if you have the exact same problem.

**Comment by cpainchaud** [comment]
Rlc-510a with very latest firmware.
